lawrencium/src/main/bash/run_job.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# This is the script to run BEAM simulation on Lawrencium cluster.
+# By-default it expects 2 input arguments: <run name> <expected execution time>,
+# though both might be manually filled in the script body.
+
+CODE_PHRASE="Execute the body of the job."
+
+# Doing shell magic - the script will send itself as a job to the cluster,
+# and first argument will tell what to do - start a job or execute the job body.
+# This way we could avoid using two shell scripts - one to start a job and another as job body.
+# if the first argument is not what we are looking for - then this shell script is used to start a job
+if [[ "$1" != "$CODE_PHRASE" ]]; then
+  echo "Starting the job .."
+
+  # what code, data and config to use for simulation
+  export BEAM_BRANCH_NAME="develop"
+  export BEAM_COMMIT_SHA
```

Comment:
Why set these to nothing? Just to leave as an option and more visible to the users? Maybe comment it out instead? More of a nit I guess, though.